src/Select/select.style.js

```diff
@@ -0,0 +1,57 @@
+import styled from "styled-components";
+import { Colors } from "../index";
+
+const OverrideSelectStyles = styled.div`
```

Comment:
As `react-select` supports/uses the `className` prop, why dont we just dervice from the select component here, that way we are not wrapping it with an extra dom element

ie. `const blah = styled(ReactSelect)`

Reply:
This "Override select styles" seems to be a mix of things that need to be unset or removed, and a mix of our own styling. 

Please remove only include unsets or "removals" (ie margin: 0) in this style part

Reply:
> Please remove only include unsets or "removals" (ie margin: 0) in this style part


so we would like to have a const for removing all the style and a const for adding our own style ?